empress/support_files/js/layouts-util.js

```diff
@@ -154,6 +154,38 @@ define(["underscore", "VectorOps", "util"], function (_, VectorOps, util) {
         return lengthGetter;
     }
 
+    /**
+     * Determines whether a given method should check if branch lengths were changed.
+     *
+     * @param {String} methodName Method for determing branch lengths.
+     *                            One of ("ultrametric", "ignore", "normal").
+     * @returns {Boolean} Indicates whether the check should be performed.
+     */
+    function shouldCheckBranchLengthsChanged(methodName) {
+        var methods = {
+            ultrametric: true,
+            ignore: true,
+            normal: false,
+        };
+        if (methodName in methods) {
+            return methods[methodName];
+        } else {
+            throw "Invalid method: '" + methodName + "'.";
+        }
+    }
+
+    var NO_LENGTHS_CHANGED_MSG =
+        "No branch lengths modified by layout options.";
```

Comment:
```suggestion
        "It doesn't look like any branch lengths were changed by the current method of modifying branch lengths.";
```

Rationale for this:

1. Gives us leeway, indicating that these messages aren't perfect (both due to tolerance things and due to missing the ignore lengths stuff i mentioned above)

2. By saying "the current method", we avoid confusing the user -- these toast messages show up every time the user changes the layout while an "alternate branch length method" (eg ultrametric) is selected, so wording things this way makes it clearer that the actual layout algorithm isn't the reason for these messages.

Reply:
(the styling code will probably want this string to be broken onto two separate lines, sorry if it yells at you)